app/controllers/application_controller.rb

```diff
@@ -1,2 +1,27 @@
 class ApplicationController < ActionController::API
+        include DeviseTokenAuth::Concerns::SetUserByToken 
+
+        rescue_from ActiveRecord::RecordNotFound, with: :render_not_found 
+        rescue_from ActiveRecord::RecordInvalid,  with: :render_record_invalid
+        rescue_from ActionController::ParameterMissing, with: :render_parameter_missing
+        rescue_from ActionDispatch::Http::Parameters::ParseError, with: :render_parse_error        
+
+        private
+
+        def render_not_found(exception)
+                render json: { error: "Not found" }, status: :not_found
```

Comment:
It would be nice if you only use 2 spaces.

Same in other places 

https://github.com/rootstrap/tech-guides/tree/master/ruby#source-code-layout